### PR TITLE
feat(optimus): [RND-166993] Add inline input error for the OptimusTextInput

### PR DESCRIPTION
**Pull request by witwash**
[RND-166993] 

#### Summary

- Added an inline error option (is using an OptimusTooltip to show it).
- Updated a story.

<details><summary>Video</summary>

CleanShot 2023-04-05 at 15 17 39

</details> 

#### Testing steps

1. Open the `Input` story
2. Set an error and enable the inline error option.
3. Test with other screen sizes.

#### Follow-up issues

None.

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

**Comment by witwash** [comment]
Look like the .gif compression ate the color of the right border 🤔 
here is the screenshot as a proof that it is still there 

<details><summary>Screenshot</summary>

CleanShot 2023-04-05 at 15 39 11

</details>

**Comment by witwash** [comment]
@ookami-kb,
ping

**Review by ookami-kb — CHANGES_REQUESTED** [review]

File: optimus/lib/src/common/field_wrapper.dart [hunk 1]
```diff
@@ -16,6 +16,7 @@ class FieldWrapper extends StatefulWidget {
     this.caption,
     this.secondaryCaption,
     this.error,
+    this.usingInlineError = false,
```
[the remark this answers is not in the document]
Comment (witwash) [reply optimus/lib/src/common/field_wrapper.dart:19]: How about `bottom` and `inline`? Or more verbose:

```
enum OptimusInputErrorVariant {
  bottomHint,
  inlineTooltip,
}
```

[the remark this answers is not in the document]
Comment (ookami-kb) [reply optimus/lib/src/common/field_wrapper.dart:19]: Yeah, both options sound fine to me 👍

File: optimus/lib/src/common/field_wrapper.dart [hunk 2]
```diff
@@ -30,6 +31,7 @@ class FieldWrapper extends StatefulWidget {
   final Widget? caption;
   final Widget? secondaryCaption;
   final String? error;
+  final bool usingInlineError;
```
Comment (ookami-kb) [inline optimus/lib/src/common/field_wrapper.dart:34]: This is still using `bool` parameter.

**Review by ookami-kb — APPROVED** [review]

**Comment by witwash** [comment]
Shoot, you were quick. I did a change to the naming

**Comment by ookami-kb** [comment]
Yeah, saw it, LGTM.

**Pull request merged**